### PR TITLE
ECL-482: Welsh translation for ECL

### PR DESCRIPTION
**Pull request by paddynski-opencast**

### CODE REVIEW

**Review by liandra-giacomine — APPROVED** [review]

**Review by liandra-giacomine — CHANGES_REQUESTED** [review]
can you remove the comments from the file

**Review by liandra-giacomine — COMMENTED** [review]
In regards to consistency, from what I can see the only other place in the code where there are comments is for our service, the comments are stated in the readme so I presume that's why they have been removed

https://github.com/search?q=repo%3Ahmrc%2Faccessibility-statement-frontend%20%22Do%20not%20include%20the%20word%20service%20at%20the%20end%22&type=code

**Review by liandra-giacomine — APPROVED** [review]

**Review by liandra-giacomine — CHANGES_REQUESTED** [review]
I discussed this with the rest of the dev team and we concluded that the comments should be removed since they're already in the readme 
https://github.com/hmrc/accessibility-statement-frontend/pull/982/files

**Comment by paddynski-opencast** [comment]
REMOVED ALL COMMENTS.

File: conf/services/economic-crime-levy.cy.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    date: 2023-10-27
+  - description: Mewnbwn ffôn ar goll ar gyfer maes rhif ffôn. Nid yw hyn yn bodloni maen prawf llwyddiant 1.3.5 (Identify Input Purpose) Canllawiau Hygyrchedd Cynnwys y We, fersiwn 2.1
+    date: 2023-10-27
+serviceLastTestedDate: 2023-06-29
+statementVisibility: public
+statementCreatedDate: 2023-08-31
+statementLastUpdatedDate: 2023-08-31
+businessArea: Customer Strategy & Tax Design (CS&TD)
+ddc: DDC Yorkshire
+liveOrClassic:
```
Comment (liandra-giacomine) [inline conf/services/economic-crime-levy.cy.yml:30]: is this supposed to be empty?

Comment (paddynski-opencast) [reply conf/services/economic-crime-levy.cy.yml:30]: It is empty in the English version of the file

**Review by liandra-giacomine — APPROVED** [review]

**Pull request merged**